lms/static/js/verify_student/views/incourse_reverify_view.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * View for in-course reverification.
+ *
+ * This view is responsible for rendering the page
+ * template, including any subviews (for photo capture).
+ */
+ var edx = edx || {};
+
+(function( $, _, Backbone, gettext ) {
+    'use strict';
+
+    edx.verify_student = edx.verify_student || {};
+
+    edx.verify_student.InCourseReverifyView = Backbone.View.extend({
+        el: '#incourse-reverify-container',
+        templateId: '#incourse_reverify-tpl',
+        submitButtonId: '#submit',
+
+        initialize: function( obj ) {
+            this.errorModel = obj.errorModel || null;
+            this.courseKey = obj.courseKey || null;
+            this.checkpointName = obj.checkpointName || null;
+
+            this.model = new edx.verify_student.ReverificationModel({
+                courseKey: this.courseKey,
+                checkpointName: this.checkpointName
+            });
+
+            this.listenTo( this.model, 'sync', _.bind( this.handleSubmitPhotoSuccess, this ));
+            this.listenTo( this.model, 'error', _.bind( this.handleSubmissionError, this ));
+        },
+
+        render: function() {
+            var renderedTemplate = _.template(
+                $( this.templateId ).html(),
+                {
+                    courseKey: this.courseKey,
+                    checkpointName: this.checkpointName
+                }
+            );
+            $( this.el ).html( renderedTemplate );
+            $( this.submitButtonId ).on( 'click', _.bind( this.submitPhoto, this ));
```

Comment:
Should be able to add this to an events hash at the top of the file e.g.
events: {
    'click #submit': 'submitPhoto'
}

For further examples of the events hash see https://github.com/edx/edx-platform/blob/1e051e5ef8c610a7bb059f8220bb8d4cb96af2ad/lms/static/js/student_account/views/LoginView.js#L14